reporter/src/main/kotlin/reporters/ExcelReporter.kt

```diff
@@ -299,21 +299,21 @@ class ExcelReporter : Reporter {
 
             val scanIssuesLines = row.scanIssues.size + row.scanIssues.flatMap { it.value }.size
 
-            sheet.createRow(currentRow).apply {
+            sheet.createRow(currentRow++).apply {
                 createCell(this, 0, row.id.toCoordinates(), font, cellStyle)
                 createCell(this, 1, scopesText, cellStyle)
                 createCell(this, 2, row.declaredLicenses.joinToString(" \n"), font, cellStyle)
                 createCell(this, 3, row.detectedLicenses.joinToString(" \n"), font, cellStyle)
                 createCell(this, 4, analyzerIssuesText, font, cellStyle)
                 createCell(this, 5, scanIssuesText, font, cellStyle)
 
-                val maxLines = listOf(
-                    scopesLines, row.declaredLicenses.size, row.detectedLicenses.size,
-                    analyzerIssuesLines, scanIssuesLines
-                ).max() ?: 1
+                val maxLines = maxOf(
+                    maxOf(scopesLines, row.declaredLicenses.size, row.detectedLicenses.size),
+                    maxOf(analyzerIssuesLines, scanIssuesLines)
```

Comment:
What are the nested `maxOf` calls for?

Reply:
Unfortunately, `maxOf()` takes at most 3 arguments. But I still felt it to be better than creating a temporary list.

Reply:
I see, but it's not really a code simplification IMO.

Reply:
Well, at least the bogus `:? 1` fallback (which was never used) is gone. Otherwise, it's probably more an optimization than a simplification, I agree.